### PR TITLE
Feature#23 branches fixes

### PR DESCRIPTION
**Pull request by EduJimenez96**
# Description
 - What?: I've fix the input type on branches/edit
 - Why?: It required an "@" to save, but it's not an email field

 - What?: I've add the destroy method to delete rows from branches table
 - Why?: It was missing in CRUD operations
 
 # Testing
-None
  
  ## Screenshots 
 -destroy

### CODE REVIEW

File: app/controllers/branches_controller.rb [hunk 1]
```diff
@@ -48,20 +48,23 @@ def update
...
-    # if @branch.active?
-    #   @branch.update(active: false)
-    # else
-    #   @branch.update(active: true)
-    # end
+  def destroy
+    @branch = Branch.find(params[:id])
+    # @branch.destroy
+
+    # redirect_to branches_path
```
Comment (Ilyeo) [inline app/controllers/branches_controller.rb:55]: please, remove these comments

File: app/views/branches/index.html.erb [hunk 2]
```diff
@@ -9,17 +9,17 @@
...
-      <% if branch.status %>
-        <tr>
-          <td><%= link_to branch.name, branch_path(branch) %></td>
-          <td><%= button_to 'Edit', edit_branch_path(branch), method: 'get', :class => "btn btn-primary"%></td>
-        </tr>
-      <% end %>
+      <tr>
+        <td><%= link_to branch.name, branch_path(branch) %></td>
+        <td><%= button_to 'Edit', edit_branch_path(branch), method: 'get', :class => "btn btn-primary"%></td>
+        <td><%= button_to 'Delete', branch_path(branch), method: 'delete', :class => "btn btn-primary"%></td>
```
Comment (Ilyeo) [inline app/views/branches/index.html.erb:21]: use new syntax and single quotes
```suggestion
        <td><%= button_to 'Edit', edit_branch_path(branch), method: 'get', class: 'btn btn-primary' %></td>
        <td><%= button_to 'Delete', branch_path(branch), method: 'delete', class: 'btn btn-primary' %></td>
```

File: db/schema.rb [hunk 3]
```diff
@@ -27,14 +27,13 @@
...
     t.datetime "updated_at", precision: 6, null: false
-    t.boolean "status", default: true
   end
 
   create_table "employees", force: :cascade do |t|
     t.string "name"
     t.string "email"
     t.string "position"
-    t.string "private_number"
+    t.integer "private_number"
```
Comment (Ilyeo) [inline db/schema.rb:36]: I think it's not a good idea to touch this file. If you need to change the type of this column you should do that through a migration, let me know your thoughts on this.

Comment (EduJimenez96) [reply db/schema.rb:36]: Oh, I don't remember touching that file, I made a migration from cmd but I will try again to see what happens

Comment (Ilyeo) [reply db/schema.rb:36]: oh yes! I can see the migration It's just above xD, my bad

**Review by Ilyeo — APPROVED** [review]
🚀

**Pull request merged**